test/base-commands/twilio-client-command.test.js

```diff
@@ -32,69 +48,68 @@ describe('base-commands', () => {
-      expect(ctx.stderr).to.contain('TWILIO_ACCOUNT_SID');
-      expect(ctx.testCmd.exit.calledWith(1)).to.be.true;
-    });
+    setUpTest(['-l', 'debug'], { setUpUserConfig: () => 0 })
+      .exit(1)
+      .it('should fail for a non-existent active project', async ctx => {
+        expect(ctx.stderr).to.contain('No project configured');
+        expect(ctx.stderr).to.contain('To add project, run: twilio projects:add');
+        expect(ctx.stderr).to.contain('TWILIO_ACCOUNT_SID');
+      });
 
-    setUpTest(['-p', 'twilio-cli-unit-testing'], userConfig => {
-      userConfig.addProject('twilio-cli-unit-testing', constants.FAKE_ACCOUNT_SID, 'stage');
-    }).it('should create a client for a non-default project', async ctx => {
-      await ctx.testCmd.run();
-      expect(ctx.testCmd.twilioClient.accountSid).to.equal(constants.FAKE_ACCOUNT_SID);
-      expect(ctx.testCmd.twilioClient.username).to.equal(constants.FAKE_API_KEY);
-      expect(ctx.testCmd.twilioClient.password).to.equal(constants.FAKE_API_SECRET + 'twilio-cli-unit-testing');
-      expect(ctx.testCmd.twilioClient.region).to.equal('stage');
-    });
+    setUpTest(['-p', 'alt', '-l', 'debug'])
+      .exit(1)
+      .it('should fail for a non-existent project', async ctx => {
+        expect(ctx.stderr).to.contain('No project configured');
+        expect(ctx.stderr).to.contain('To add project, run: twilio projects:add -p alt');
+        expect(ctx.stderr).to.contain('TWILIO_ACCOUNT_SID');
+      });
 
-    setUpTest(
-      ['-p', 'twilio-cli-unit-testing'],
-      userConfig => {
-        userConfig.addProject('twilio-cli-unit-testing', constants.FAKE_ACCOUNT_SID);
-      },
-      false
-    ).it('should handle a secure storage error', async ctx => {
-      ctx.testCmd.exit = sinon.fake();
-      await ctx.testCmd.run();
-      expect(ctx.stderr).to.contain('Could not get credentials for project "twilio-cli-unit-testing"');
-      expect(ctx.stderr).to.contain('To reconfigure project, run: twilio projects:add -p twilio-cli-unit-testing');
-      expect(ctx.testCmd.exit.calledWith(1)).to.be.true;
-    });
+    setUpTest(['-p', 'twilio-cli-unit-testing'])
+      .it('should create a client for a non-default project', async ctx => {
+        expect(ctx.testCmd.twilioClient.accountSid).to.equal(constants.FAKE_ACCOUNT_SID);
+        expect(ctx.testCmd.twilioClient.username).to.equal(constants.FAKE_API_KEY);
+        expect(ctx.testCmd.twilioClient.password).to.equal(constants.FAKE_API_SECRET + 'twilio-cli-unit-testing');
+        expect(ctx.testCmd.twilioClient.region).to.equal('stage');
+      });
+
+    setUpTest(['-p', 'twilio-cli-unit-testing'], { mockSecureStorage: false })
+      .exit(1)
+      .it('should handle a secure storage error', async ctx => {
+        expect(ctx.stderr).to.contain('Could not get credentials for project "twilio-cli-unit-testing"');
+        expect(ctx.stderr).to.contain('To reconfigure project, run: twilio projects:add -p twilio-cli-unit-testing');
```

Comment:
not: To reconfigure project -> To reconfigure the project or To reconfigure your project

Reply:
I like the former.